wishlist.txt

```diff
@@ -161,3 +161,7 @@ Render / TexerII
     * Rotate sprites. ("r" per-point variable?)
     * Why not "rz" right away, and add "rx" & "ry" to skew the image? Which perspective to use?
     * Bug: image filenames starting with "_" are buggy (because they are sorted before the default image).
+
+Input / Audio handling
+    * Enable equalizer-/compressor-effects on input spectrum-data (implement in sdl-audiocallback)
```

Comment:
Please use two empty lines before the heading and one empty line after, thanks! 🙂 

You can remove the implementation hint, it's not needed in this file (the tech used is not really relevant to the wishlist).